webserver/observe/metrics/src/main/java/io/helidon/webserver/observe/metrics/KeyPerformanceIndicatorMetricsImpls.java

```diff
@@ -29,31 +31,26 @@
 
 class KeyPerformanceIndicatorMetricsImpls {
 
+    /**
+     * Name for metric recording number of requests currently deferred (received but not yet processing).
+     */
+    public static final String DEFERRED_NAME = "deferred";
```

Comment:
`public` is redundant here, as this is a package private class.

Reply:
changed